openmic/util.py

```diff
@@ -0,0 +1,48 @@
+import hashlib
```

Comment:
should have a docstring here

Reply:
sidebar from IRL: At least a one-line docstring of what this is for. encoding and shebang are optional but encouraged.